deployment/gunicorn/commcarehq_wsgi.py

```diff
@@ -15,5 +15,8 @@
 patch_all(subprocess=True)
 patch_psycopg()
 
+from manage import run_patches
```

Comment:
Can `run_patches` be imported above on line 3? (if it's needed at all, see below)